static/css/custom.css

```diff
@@ -6,3 +6,16 @@
     width: 400px;
 }
 }
+
+p, li {
+    line-height: 1.7em;
+}
```

Comment:
this shouldn't be part of this PR but I don't have time to do another PR and rebase this one so...